docs/using-docker.rst

```diff
@@ -61,16 +59,15 @@ We provide an example compose file (``containers/docker-compose.yml.example``).
       teleirc:
         user: teleirc
         build:
-          context: .
-          dockerfile: Dockerfile.alpine
+          context: ..
+          dockerfile: containers/Dockerfile.alpine
         env_file: .env
 
-We ignore the ``docker-compose.yml`` file in ``.gitignore``.
-
 Running with Compose
 ====================
 
 Run these commands to begin using Teleirc with Docker Compose.
 
-#. Copy ``containers/docker-compose.yml.example`` to ``docker-compose.yml``
```

Comment:
I understood the change for the `config.js`, but I'm not sure I understand it for the docker-compose file. If a user makes changes to the docker-compose file, shouldn't we respect user preference? This might make in-place upgrading more annoying. I'm not sure I understand the benefit of this change.

Reply:
I can revert this if you want it reverted...I just found it silly -- also after pushing the images -- we won't have to worry about the implications of in-place upgrades

Reply:
Reverted.

Reply:
@robbyoconnor I see what you mean now. We can revisit once we have public images pushed. It might be some time before I can get to it, and over in RITlug, we were talking about trying to set up some task / easy fix bounties in our community to encourage some new contributors. This might be one of those things. :slightly_smiling_face:

Reply:
:+1: